auth.go

```diff
@@ -444,6 +452,39 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 	return sessionkey, token, nil
 }
 
+func auditUserAction(identity, password, actionDescription string) error {
+	auditServiceUrl := "http://localhost:2016/auth-logaction"
```

Comment:
I agree that it's too big. We should make a file called "audit.go", with this stuff inside it. Also, I suggested this morning to create an Auditor interface. It can be extremely simple - it just needs to take the parameters that auditUserAction takes. But we *must* do that, in order to preserve the property that authaus doesn't explicitly depend on any IMQS services.
It must also be optional - ie if the Auditor is null, then auditUserAction just returns without doing anything.